test/unit/runner.spec.js

```diff
@@ -452,9 +451,29 @@ describe('Runner', function() {
       'at processImmediate [as _immediateCallback] (timers.js:321:17)'
     ];
 
+    before(function() {
+      // Only for Node running on Windows
+      if ('platform' in process && process.platform === 'win32') {
```

Comment:
`'platform' in process` is unnecessary check

Reply:
communicates the fact that it _may not exist_.